extensions/setup/index.json

```diff
@@ -20,6 +20,7 @@
   ],
   "beta": [
     "seo",
+    "testimonial",
```

Comment:
Please rebase; as of https://github.com/Automattic/jetpack/pull/11971 this file has been moved to `extensions/index.json` from `setup`.